easyaudit/migrations/0011_auto_20181101_1339.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-11-01 13:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('easyaudit', '0010_repr_text'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='requestevent',
+            name='url',
+            field=models.TextField(),
```

Comment:
Actually I think this migration actually explicitly undoes the previous ones index. So that way the project is consistent.

Reply:
Awesome. Any objections to making this column a `CharField`? If it doesn't create problems for you, it'd prevent conflicts with a PR I'm currently working on: https://github.com/soynatan/django-easy-audit/pull/82/files

Reply:
I can't recall why we changed the URL to be a `TextField`. But I believe we should make it a `CharField` again. @jheld what do you think? Am I missing something? Or maybe we should leave it a `TextField` and make this change afterwards.

Reply:
@soynatan If you look at my PR #82 , I change the field to a URL.

Reply:
I agree charfield would be best. From the other comment, I am more inclined to splitting up url and query params to separate fields at a later date, and change the type then.

Reply:
> I am more inclined to splitting up url and query params to separate fields at a later date

Maybe I'm missing something (I've been away for some time) but, we already have url and query params on separate fields.